src/Process/Listener/MutationTestingConsoleLoggerSubscriber.php

```diff
@@ -95,18 +95,28 @@ public function onMutationTestingFinished(MutationTestingFinished $event)
     {
         // TODO [doc] write test -> run mutation for just this file. Should be 100%, 100%, 100%,
         $this->outputFormatter->finish();
-        $processes = $this->metricsCalculator->getEscapedMutantProcesses();
 
         if ($this->showMutations) {
```

Comment:
would it better to move the second condition with verbosity under this one?

```php
if ($this->showMutations) {
    // do something

    if ($this->output->getVerbosity() > OutputInterface::VERBOSITY_NORMAL) {
       // do something else
    }
}
```

Reply:
You are right. One moment!